src/pages/components/externalPages/TestimonialExternalPages/TestimonialContentTemplate.js

```diff
@@ -2,18 +2,26 @@ import React, { Fragment } from 'react';
 import { LazyLoadImage } from 'react-lazy-load-image-component';
 import TestimonialMainContent from './TestimonialMainContent';
 
-const TestimonialContentTemplate = ({ description }) => (
+const TestimonialContentTemplate = ({ mode, description }) => (
   <Fragment>
-    <header className='flex px-[20px] pt-[64px] sm:px-[55px] sm:pt-[38px] lg:px-[100px] lg:pt-[51px] xl:px-[200px] 2xl:px-[339px]'>
-      <LazyLoadImage src='/images/Avatar01.svg' alt='profile' className='h-[71px]' />
+    <header
+      className={`${
+        mode === 'light' ? '' : 'bg-dark-ash-800'
```

Comment:
```suggestion
        dark:bg-dark-ash-800
```
I think you make it like this